### PR TITLE
Evil promise

### PR DESCRIPTION
**Pull request by Sneezoo**

### CODE REVIEW

File: client/app/common/user/user.service.js [hunk 1]
```diff
@@ -13,42 +10,19 @@ class UserService extends base {
...
   }
 
-  save(id, updates) {
-    if (UserService.validate(["id"])) {
-      return this.$http.patch(`/api/users/${id}/`, updates).then((res) => res.data);
-    } else {
-      return Promise.reject();
-    }
+  save(user) {
+    return this.$http.patch(`/api/users/${user.id}/`, user).then((res) => res.data);
```
Comment (Xbaal) [inline client/app/common/user/user.service.js]: All the changes are already merged, as far as i saw it.
So, i will delete the pull request and branch.

**Comment by tiltec** [comment]
@Sneezoo I couldn't merge this branch, as there were things to fix and I didn't have time to fix them. Meanwhile, @Xbaal adjusted the test style and removed the promises while doing that. I'm quite happy that we are so productive now and I guess things like this will occur with less probability when we are more used to working collaboratively.

**Comment by Xbaal** [comment]
@Sneezoo like Tilmann said, the branch was not mergeable. As far as i can remember, i didn't see an easy way to resolve the problem, so i decided to work on other tasks and by that i happended to do your changes again.
I have been quite new to all this working together and using GitHub and knowing how your working style is.

**Comment by Sneezoo** [comment]
@Xbaal @tiltec This comment was just to remind you. So this doesn't happen anymore.

**Pull request closed without merging**